internal/services/datalake/data_lake_store_resource_test.go

```diff
@@ -35,6 +35,24 @@ func TestAccDataLakeStore_basic(t *testing.T) {
 	})
 }
 
+func TestAccDataLakeStore_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_data_lake_store", "test")
+	r := DataLakeStoreResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("tier").HasValue("Consumption"),
+				check.That(data.ResourceName).Key("encryption_state").HasValue("Enabled"),
+				check.That(data.ResourceName).Key("encryption_type").HasValue("ServiceManaged"),
```

Comment:
Not necessary to add these checks, except `ExistsInAzure`

Reply:
Got it. Thanks!